static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/SubscriptionDetails.tsx

```diff
@@ -1,18 +1,28 @@
+import { Button } from "@canonical/react-components";
 import React from "react";
 
-const SubscriptionDetails = () => (
-  <div className="p-subscribe__details">
-    <p>Details</p>
-    <p>Details</p>
-    <p>Details</p>
-    <p>Details</p>
-    <p>Details</p>
-    <p>Details</p>
-    <p>Details</p>
-    <p>Details</p>
-    <p>Details</p>
-    <p>Details</p>
-  </div>
-);
+import DetailsContent from "./DetailsTabs";
```

Comment:
I think you're importing the wrong one here

Reply:
Hah, not sure how I managed that, maybe when I rebased or something.